packages/revocation/contracts/RevocationRegistry.sol

```diff
@@ -0,0 +1,31 @@
+pragma solidity >=0.7.0 <0.9.0;
+
+/**
+* A contract for tracking the revocations of off-chain verifiable credentials.
+* For on-chain role revocations, see contract in iam-contracts repository.
+*/  
+contract RevocationRegistry {
+    
+    struct RevokedClaim {
+        address revoker;
+        uint revokedTimestamp;
+    }
+    mapping (bytes32 => RevokedClaim[]) revokedClaims;
+ 
+    function revokeClaim(bytes32 credential) public  {
```

Comment:
Could you please document what credentials are?

Reply:
Would add a comment.